app/javascript/utilities/textAreaUtils.js

```diff
@@ -111,3 +114,42 @@ const getIndexOfCurrentWordAutocompleteSymbol = (content, selectionIndex) => {
 
   return -1;
 };
+
+/**
+ * This hook can be used to keep the height of a textarea in step with the current content height, avoiding a scrolling textarea.
+ * An optional array of additional elements can be set. If provided, all additional elements will receive the same height.
+ *
+ * @example
+ *
+ * const { setTextArea } = useTextAreaAutoResize();
+ * setTextArea(myTextAreaRef.current);
+ * setAdditionalElements([myOtherElement.current]);
+ */
+export const useTextAreaAutoResize = () => {
+  const [textArea, setTextArea] = useState(null);
+  const [additionalElements, setAdditionalElements] = useState([]);
```

Comment:
This is so that the two text areas (combobox / normal textarea) stay in step at the same time. Auto-resizing them separately caused small layout shifts. The `additionalElements` is totally optional and will probably only ever be used by the mention autocomplete component